### DISCUSSION TITLE
What exactly does expansion_autoscale do?

### DISCUSSION THREAD

**Discussion by Lyserius** [induktio/thinker#74] [Q&A]
I'm confused since the AI sometimes has more bases than me, so how exactly does the expansion_autoscale setting work?

**Comment by induktio** [comment]
If expansion_autoscale is enabled, it simply causes the AI base limit to be one of following, whichever is highest: expansion_limit and player base count. If it is disabled, the base limit is always the same as expansion_limit. By default these limits are set fairly high though.
